src/index.ts

```diff
@@ -176,3 +179,25 @@ async function statusOk(res: Response, logger: Logger): Promise<boolean> {
     logger.debug('testing response for whether it is "ok". has status: ', res.status, ' debug: ', JSON.stringify(res))
     return String(res.status)[0] === '2'
 }
+
+function getProperties(event: PluginEvent, { config }: SalesforcePluginMeta): Properties {
+    // Spreading so the TypeScript compiler understands that in the
+    // reducer there's no way the properties will be undefined
+    const { properties } = event
+
+    if (!properties) return {}
+    if (!config.parametersToInclude) return properties
+
+    const allParameters = config.parametersToInclude.split(',')
```

Comment:
These should ideally be `.trim()`'ed too, in case of values like these `foo, bar`